packages/dart/sshnoports/bin/npp_atserver.dart

```diff
@@ -119,6 +120,23 @@ void main(List<String> args) async {
     logger.info('daemonAtSigns is now ${sshnpa.daemonAtsigns}');
   });
 
+  // start updating the heartbeat atkey periodically
+  Timer.periodic(const Duration(seconds: 60), (_) async {
+    await _updateHeartbeatKey(atClient); // key format: `heartbeat.noports@<atsign>`: {'timestamp': '...'}
+  });
+
+  // start listening for force heartbeats from the same atSign
+  logger.shout('Starting AtRpc Server to listen for forced heartbeats...');
+  AtRpc(
+    atClient: atClient,
+    baseNameSpace: 'sshnp.noports',
```

Comment:
@gkc @XavierChanth 

The namespace for NoPorts Desktop is `noports` but for npp_atserver it is `sshnp`

These two need to RPC to each other. What should the baseNameSpace be?